lib/util/token_or_null.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+import '../stores/accounts_store.dart';
+
+/// returns either raw token for default user of `instanceUrl`
+/// or null if this instance is not added or doesn't
+/// have any user logged in
+String tokenOrNull(BuildContext context, String instanceUrl) {
```

Comment:
does not handle the case where someone does not have this instance added

Please modify `AccountsStore.isAnonymousFor` in a way where it checks if the instance is added (which would then mean it is anonymous)